packages/rest/src/keys.ts

```diff
@@ -5,7 +5,12 @@
 
 import {CoreBindings} from '@loopback/core';
 import {BindingKey, Context} from '@loopback/context';
-import {OpenApiSpec} from '@loopback/openapi-v3-types';
+
+/**
+ * See https://github.com/Microsoft/TypeScript/issues/26985
+ */
+// import {OpenApiSpec} from '@loopback/openapi-v3-types';
```

Comment:
can we remove this import statement?

Reply:
I intentionally kept it here so that we don't forget to revert it once the TS issue is fixed.